src/pages/game/[id].tsx

```diff
@@ -1,135 +1,94 @@
+          <ButtonComponent variant="Secondary" ><BsWifi /></ButtonComponent>
+          <ButtonComponent variant="Secondary" ><BsWifiOff /></ButtonComponent>
+          <input placeholder="E.g 'Base pack'" />
+        </form>
+
+        <hr />
+
+        {/* List of addons */}
+        <section className={styles.addons} onScroll={scrolling}>
+          {addons.data?.pages.map(page =>
+            page.items.map(x =>
+              <AddonComponent key={x.id} addon={x} />
+            )
+          )
+          }
+        </section>
+
+        {/* Card count and settings */}
+        <section className={styles.horizontalList}>
+          <TagComponent><BsWifi />125</TagComponent>
+          <TagComponent><BsWifiOff />125</TagComponent>
+          <ButtonComponent variant="Secondary">Settings</ButtonComponent>
+        </section>
+
+        {/* Start */}
+        <section className={styles.horizontalList}>
+          <ButtonComponent>Start Online</ButtonComponent>
+          <ButtonComponent>Start Offline</ButtonComponent>
+        </section>
+
+      </main>
     </LayoutComponent>
   );
-
-  return <div>Error</div>;
 };
 
-export default GamePage;
-
-
 interface Props {
-  game: Game;
-  addons: Addon[];
+  game: Game
 }
 
+export default GameSetup;
 
-export async function getStaticPaths() {
-  const games = await prisma.game.findMany();
-  const paths = games.map(game => ({ params: { id: game.id } }));
-
+export const getStaticPaths: GetStaticPaths = async () => {
+  const result = await prisma.game.findMany();
+  const paths = result.map((project) => ({ params: { id: project.id } }));
   return {
-    paths,
-    fallback: false,
+    paths: paths,
+    fallback: "blocking",
   };
-}
-
-export async function getStaticProps(context: GetStaticPropsContext) {
-  const gameResult = await prisma.game.findUnique({ where: { id: context.params?.id as string } });
-  if (gameResult === null) return { notFound: true };
-  const game = { ...gameResult, created_at: Number(gameResult.created_at) };
-
-  const addonResult = await prisma.addon.findMany({
-    where: { game_id: gameResult.id },
-    include: { _count: { select: { cards: true } }, authors: true, cards: true },
-  });
-
-  const addons: Addon[] = addonResult.map(addon => {
-    return {
-      ...addon,
-      created_at: Number(addon.created_at),
-      _count: null,
-      card_count: addon._count.cards,
-      authors: addon.authors.map(x => x.name),
-      cards: addon.cards.map(card => ({ ...card, created_at: Number(addon.created_at) })),
-    };
-  }
-  ).filter(addon => addon.card_count > 1);
+};
 
-  return {
-    props: { game, addons },
+export const getStaticProps: GetStaticProps = async ({ params }) => {
+  const result = await prisma.game.findUnique({ where: { id: params?.id as string } });
+  if (!result) return { notFound: true };
+  const game = {
+    ...result,
+    created_at: Number(result.created_at),
+    updated_at: Number(result.updated_at),
   };
```

Comment:
I've seen this conversion done at least once elsewhere. Might be good to extract a helper which takes in an object with both `created_at` and `updated_at`, and performs the number conversion.

Something like 
```ts
const withTimestampsAsNumber = <T extends { created_at: Date; updated_at: Date }>(
  data: T,
): Omit<T, "created_at" | "updated_at"> & { created_at: number; updated_at: number } => {
  return {
    ...data,
    created_at: Number(data.created_at),
    updated_at: Number(data.updated_at),
  };
};
```

or, if you want to go a more generic route:

```ts
const castFieldsToNumber = <T extends Record<string, unknown>, K extends keyof T>(
  data: T,
  fields: K[]
): Omit<T, K> & { [Key in K]: number } => {
  const clone = { ...data };
  fields.forEach((field) => {
    clone[field] = Number(clone[field]) as T[K];
  });

  // We need the cast here as the type system doesn't know that have casted the fields to number
  return clone as Omit<T, K> & { [Key in K]: number };
};

const withTimestampsAsNumber = <T extends { created_at: Date; updated_at: Date }>(data: T) =>
  castFieldsToNumber(data, ["created_at", "updated_at"]);
```